Dockerfile

```diff
@@ -1,13 +1,17 @@
 # Build stage
 FROM node:20 AS build
 WORKDIR /app
-COPY package*.json ./
-RUN npm ci
+COPY package.json ./
+RUN npm install
```

Comment:
Why the move from `npm ci` to `npm install`? Did you need to update dependencies or troubleshoot something? In Docker, `npm ci` is typically the go-to because it’s faster, locks in exact versions from `package-lock.json`, and ensures a clean, consistent build every time. Any particular reason for switching?

Reply:
Yeah, we tried ci and it was producing errors so we switched to npm install.